pkg/webui/console/components/device-data-form/validation-schema.js

```diff
@@ -73,6 +73,12 @@ export default Yup.object().shape({
   lorawan_phy_version: Yup.string().required(sharedMessages.validateRequired),
   frequency_plan_id: Yup.string().required(sharedMessages.validateRequired),
   supports_class_c: Yup.boolean(),
+  network_server_address: Yup.string()
+    .matches(addressRegexp, sharedMessages.validateAddressFormat),
```

Comment:
The validation message also needs to be adapted then, it is still saying to use `host:port`